docs/examples/bitops-deploy-heyemoji/test/terraform/providers.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
     required_version = ">= 0.12"
     backend "s3" {
-        bucket = "heyemoji-blog"
+        bucket = "heyemoji_blog"
```

Comment:
Underscores are not allowed for S3 Buckets

Reply:
Thanks! 👍  

Fixed in https://github.com/bitovi/bitops/pull/364/commits/f5ca7493b2e02258e39291d89399267f6925d10a